### ISSUE TITLE
add auto import of files

### ISSUE THREAD

**Issue by lucki1000** [beemdevelopment/Aegis#938]
Hi, 

I find it useful to have a auto-import function so i can sync between my devices over a local storage.
Actually i use the auto backup function but have to restore any item on my other device everytime i changed something.
The App is really nice and it would be great to see more function in the future.

Regards
Lucki1000

**Comment by kusyka911** [comment]
Agree, file-based sync is nice feature. I'm also using a notes app with file based sync.
